tap_shopify/__init__.py

```diff
@@ -206,6 +206,8 @@ def main():
             msg = body.get('errors')
         finally:
             raise ShopifyError(exc, msg) from exc
+    except ShopifyError as error:
+        raise error
     except Exception as exc:
         raise ShopifyError(exc) from exc
```

Comment:
is this needed?

Reply:
Yes, to display the custom exception message. Raising the ShopifyError with all the necessary details as Shopify x-request-id, HTTP status code, and the stream name, will help the customers to reach out to the Shopify support and get the issue resolved.